src/ResultWriter/EnergyOutput.cpp

```diff
@@ -519,6 +565,36 @@ void EnergyOutput::printEnergies() {
     }
   }
 }
+void EnergyOutput::checkAbortCriterion() {
+  const auto rank = MPI::mpi.rank();
+  bool abort = false;
+  if (rank == 0) {
+    if ((minTimeSinceSlipRateBelowThreshold > 0) and
+        (minTimeSinceSlipRateBelowThreshold < std::numeric_limits<real>::max())) {
+      if (static_cast<double>(minTimeSinceSlipRateBelowThreshold) < terminatorMaxTimePostRupture) {
+        logInfo(rank) << "all slip rates are below threshold since"
+                      << minTimeSinceSlipRateBelowThreshold
+                      << "s (lower than the abort criteria: " << terminatorMaxTimePostRupture
+                      << "s)";
+      } else {
+        logInfo(rank) << "all slip rates are below threshold since"
+                      << minTimeSinceSlipRateBelowThreshold
+                      << "s (greater than the abort criteria: " << terminatorMaxTimePostRupture
+                      << "s)";
+        logInfo(rank) << "aborting...";
```

Comment:
why not use a `logError()` here?

Reply:
because it is not an error, but a controlled way of stopping seissol.
Let say I run several setups with this bash script:
```
#!/bin/bash
set -euov pipefail
   for par_file in parameters_dyn*; do
       echo "Processing file: $par_file"
       mpirun -n 2 --map-by ppr:1:numa:pe=10 --report-bindings seissol-launch SeisSol_Release_ssm_86_cuda_4_elastic $par_file  2>&1 | tee logs/$par_file.out
   done
```
Then my bash script will stop with logError and not process to the next parameter file.